provider/cluster/service.go

```diff
@@ -95,7 +89,7 @@ type service struct {
 
 	inventory *inventoryService
 
-	statusch  chan chan<- *Status
+	statusch  chan chan<- *ctypes.Status
```

Comment:
maybe for later. change to value, not pointer. pointer creates some pressure on GC

Reply:
I will add an issue to address this in the future.